frontend/src/core/scaffold/views/scaffoldAssetView.js

```diff
@@ -284,6 +284,7 @@ define(function(require) {
 
             // Check if alternative attribute should be used
             if (alternativeAttribute) {
+                attributesToSave = [];
```

Comment:
should `attributesToSave` only be initialized if it doesn't already exist? Can it exist at this point?